src/registry/select/edit-validation.ts

```diff
@@ -18,7 +18,11 @@ const buildValuesSchema = (intl: IntlShape) =>
       values: optionSchema(intl).array().min(1).optional(),
     }),
     openForms: z.object({
-      dataSrc: z.union([z.literal('manual'), z.literal('variable')]),
+      dataSrc: z.union([
+        z.literal('manual'),
+        z.literal('variable'),
+        z.literal('referentielijsten'),
+      ]),
```

Comment:
same remark about missing `code` and `service` (similar to the `itemsExpression`)